www/front_src/src/Header/index.tsx

```diff
@@ -1,28 +1,30 @@
 import React from 'react';
 
 import classnames from 'classnames';
-import { useAtomValue } from 'jotai/utils';
-
-import { refreshIntervalAtom } from '@centreon/ui-context';
 
 import Hook from '../components/Hook';
 
 import styles from './header.scss';
-import PollerMenu from './pollerMenu';
+import PollerMenu, { pollerConfigurationNumberPage } from './PollerMenu';
 import UserMenu from './userMenu';
 import ServiceStatusCounter from './RessourceStatusCounter/Service';
 import HostStatusCounter from './RessourceStatusCounter/Host';
 
 const HookComponent = Hook as unknown as (props) => JSX.Element;
 
 const Header = (): JSX.Element => {
-  const refreshInterval = useAtomValue(refreshIntervalAtom);
+  const pollerListIssues =
+    'internal.php?object=centreon_topcounter&action=pollersListIssues';
 
   return (
     <header className={styles.header}>
       <div className={styles['header-icons']}>
         <div className={classnames(styles.wrap, styles['wrap-left'])}>
-          <PollerMenu refreshInterval={refreshInterval} />
+          <PollerMenu
+            allowedPages={[pollerConfigurationNumberPage]}
```

Comment:
```suggestion
```
As allowedPages is provided by Redux through the `connect` + `mapStateToProps`, you don't need this